meta/flag-references/de.md

```diff
@@ -0,0 +1,237 @@
+[flag_MG]: /wiki/shared/flag/MG.gif "Madagaskar"
+[flag_MH]: /wiki/shared/flag/MH.gif "Marshallinseln"
+[flag_MK]: /wiki/shared/flag/MK.gif "Nordmazedonien"
+[flag_ML]: /wiki/shared/flag/ML.gif "Mali"
+[flag_MM]: /wiki/shared/flag/MM.gif "Myanmar"
+[flag_MN]: /wiki/shared/flag/MN.gif "Mongolei"
+[flag_MO]: /wiki/shared/flag/MO.gif "Macau"
+[flag_MP]: /wiki/shared/flag/MP.gif "Nördliche Marianen"
+[flag_MQ]: /wiki/shared/flag/MQ.gif "Martinique"
+[flag_MR]: /wiki/shared/flag/MR.gif "Mauretanien"
+[flag_MS]: /wiki/shared/flag/MS.gif "Montserrat"
+[flag_MT]: /wiki/shared/flag/MT.gif "Malta"
+[flag_MU]: /wiki/shared/flag/MU.gif "Mauritius"
+[flag_MV]: /wiki/shared/flag/MV.gif "Malediven"
+[flag_MW]: /wiki/shared/flag/MW.gif "Malawi"
+[flag_MX]: /wiki/shared/flag/MX.gif "Mexiko"
+[flag_MY]: /wiki/shared/flag/MY.gif "Malaysia"
+[flag_MZ]: /wiki/shared/flag/MZ.gif "Mosambik"
+[flag_NA]: /wiki/shared/flag/NA.gif "Namibia"
+[flag_NC]: /wiki/shared/flag/NC.gif "Neukaledonien"
+[flag_NE]: /wiki/shared/flag/NE.gif "Niger"
+[flag_NF]: /wiki/shared/flag/NF.gif "Norfolkinsel"
+[flag_NG]: /wiki/shared/flag/NG.gif "Nigeria"
+[flag_NI]: /wiki/shared/flag/NI.gif "Nicaragua"
+[flag_NL]: /wiki/shared/flag/NL.gif "Niederlande"
+[flag_NO]: /wiki/shared/flag/NO.gif "Norwegen"
+[flag_NP]: /wiki/shared/flag/NP.gif "Nepal"
+[flag_NR]: /wiki/shared/flag/NR.gif "Nauru"
+[flag_NU]: /wiki/shared/flag/NU.gif "Niue"
+[flag_NZ]: /wiki/shared/flag/NZ.gif "Neuseeland"
+[flag_OM]: /wiki/shared/flag/OM.gif "Oman"
+[flag_PA]: /wiki/shared/flag/PA.gif "Panama"
+[flag_PE]: /wiki/shared/flag/PE.gif "Peru"
+[flag_PF]: /wiki/shared/flag/PF.gif "Französisch-Polynesien"
+[flag_PG]: /wiki/shared/flag/PG.gif "Papua-Neuguinea"
+[flag_PH]: /wiki/shared/flag/PH.gif "Philippinen"
+[flag_PK]: /wiki/shared/flag/PK.gif "Pakistan"
+[flag_PL]: /wiki/shared/flag/PL.gif "Polen"
+[flag_PM]: /wiki/shared/flag/PM.gif "Saint-Pierre und Miquelon"
+[flag_PN]: /wiki/shared/flag/PN.gif "Pitcairn"
+[flag_PR]: /wiki/shared/flag/PR.gif "Puerto Rico"
+[flag_PS]: /wiki/shared/flag/PS.gif "Palästinensische Autonomiegebiete"
+[flag_PT]: /wiki/shared/flag/PT.gif "Portugal"
+[flag_PW]: /wiki/shared/flag/PW.gif "Palau"
+[flag_PY]: /wiki/shared/flag/PY.gif "Paraguay"
+[flag_QA]: /wiki/shared/flag/QA.gif "Katar"
+[flag_RE]: /wiki/shared/flag/RE.gif "Reunion"
+[flag_RO]: /wiki/shared/flag/RO.gif "Rumänien"
+[flag_RS]: /wiki/shared/flag/RS.gif "Serbien"
+[flag_RU]: /wiki/shared/flag/RU.gif "Russische Föderation"
+[flag_RW]: /wiki/shared/flag/RW.gif "Ruanda"
+[flag_SA]: /wiki/shared/flag/SA.gif "Saudi-Arabien"
+[flag_SB]: /wiki/shared/flag/SB.gif "Salomonen"
+[flag_SC]: /wiki/shared/flag/SC.gif "Seychellen"
+[flag_SD]: /wiki/shared/flag/SD.gif "Sudan"
+[flag_SE]: /wiki/shared/flag/SE.gif "Schweden"
+[flag_SG]: /wiki/shared/flag/SG.gif "Singapur"
+[flag_SH]: /wiki/shared/flag/SH.gif "St. Helena"
+[flag_SI]: /wiki/shared/flag/SI.gif "Slowenien"
+[flag_SJ]: /wiki/shared/flag/SJ.gif "Svalbard und Jan Mayen"
```

Comment:
```suggestion
[flag_SJ]: /wiki/shared/flag/SJ.gif "Spitzbergen und Jan Mayen"
```
apparently both is fine? taking [this](https://de.wikipedia.org/wiki/ISO-3166-1-Kodierliste) as reference but when you click on the `SJ` link its written like yours so idk lol

Reply:
`Svalbard` might be a more international term, so I will use this.